{{ cookiecutter.folder_name }}/docs/source/conf.py

```diff
@@ -15,13 +15,15 @@
 import os
 import sys
 import sphinx_rtd_theme
-module_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),'../../')
-sys.path.insert(0,module_path)
+
+module_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                           '../../')
+sys.path.insert(0, module_path)
 
 # -- Project information -----------------------------------------------------
 
 project = '{{ cookiecutter.project_name }}'
-copyright = ('2018, {}'.format({{ cookiecutter.author_name }}))
+copyright = '2019, {{ cookiecutter.author_name }}'
```

Comment:
Would it be worthwhile to make the year auto-update to the year during which the docs were built? I'm not sure how this sort of thing should work but it could prevent us from doing celebratory New year's PR's to update *all* of our packages' licenses. The same goes for the license file.

Reply:
If it's something that should be set only when the package is created, we could probably add a pre-build hook.sh to set a 'year' variable for insertion into the `conf.py` and the license file.

Reply:
This is done automatically now, thanks @teddyrendahl!